Griddly/Views/Shared/Griddly/GriddlyFilters.cshtml

```diff
@@ -40,7 +40,18 @@ else
                 else
                 {
                     if (!css.Is(CssFramework.Bootstrap3)) { <h4 class="modal-title">Filter</h4> }
-                    <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+
+                    if (css.Is(CssFramework.Bootstrap5))
+                    {
+                        <button type="button" class="close btn-close-custom order-3 ms-2" data-dismiss="modal" bs-data-dismiss="modal" aria-label="Close">
```

Comment:
Shouldn't that be `data-bs-dismiss`? Also, use the standard `btn-close` instead of `btn-close-custom` and no need for `data-dismiss` or the other styling classes. Those should be applied with project specific stylesheets.

```suggestion
                        <button type="button" class="close btn-close" data-bs-dismiss="modal" aria-label="Close">
```